packages/ebayui-core/src/components/ebay-filter-chip/index.marko

```diff
@@ -0,0 +1,51 @@
+import { processHtmlAttributes } from "../../common/html-attributes";
+$ const {
+    renderBody,
+    disabled,
+    selected,
+    expanded,
+    icon,
+    image,
+    clippedText = "Filter Applied",
```

Comment:
Should it be called `a11ySelectedText` instead for consistency with others accessibility texts?